internal/app/wwctl/profile/set/root.go

```diff
@@ -118,6 +118,7 @@ func init() {
 
 	baseCmd.PersistentFlags().StringVarP(&SetNetName, "netname", "n", "default", "Define the network name to configure")
 	baseCmd.PersistentFlags().StringVarP(&SetNetDev, "netdev", "N", "", "Set the node's network device")
+	baseCmd.PersistentFlags().StringVar(&SetNetDefault, "primary", "", "Enable/disable device as primary (yes/no)")
```

Comment:
I don't believe this is correct. It should still be `--default` not primary, as this will change the NetDev.Default struct member.

Reply:
I didn't change the internal struct member, as regular users don't see or use it. This changes only the command option and YAML tag.
 
The internal struct should also be updated, but it will be lot more changes and I wanted to wait until the new API is implemented.

Reply:
Sorry, that's not what I meant.

the `SetNetDefault` variable triggers the `DEFAULT=...` in the ifcfg-* templates. Meaning, that has nothing to do with "primary".